cachelib/adaptor/rocks_secondary_cache/CachelibWrapper.cpp

```diff
@@ -225,64 +286,31 @@ void RocksCachelibWrapper::Close() {
     // sections already started to finish, and then delete the cache
     cache_.store(nullptr);
     GetRcuDomain().synchronize();
-    admin_.reset();
     delete cache;
   }
 }
 
-bool RocksCachelibWrapper::UpdateMaxWriteRateForDynamicRandom(
```

Comment:
Can this not be supported anymore? Internally, we need this API.

Reply:
@anand1976 This is a feature that is internal to Facebook/Meta.  The header file that is required for this is not part of the public API.